quickstart/inline-caching/cache-loader/src/main/java/io/vmware/event/ItemCacheLoader.java

```diff
@@ -0,0 +1,56 @@
+package io.vmware.event;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+import org.jooq.DSLContext;
+import org.jooq.Record;
+import org.jooq.Result;
+import org.jooq.SQLDialect;
+import org.jooq.impl.DSL;
+
+import org.apache.geode.LogWriter;
+import org.apache.geode.cache.CacheFactory;
+import org.apache.geode.cache.CacheLoader;
+import org.apache.geode.cache.LoaderHelper;
+
+@SuppressWarnings({ "deprecation" })
+public class ItemCacheLoader implements CacheLoader<String, String> {
+	
+	private static LogWriter log;
+
+	static {
+		log = CacheFactory.getAnyInstance().getDistributedSystem().getLogWriter();
+	}
+
+	public String load(LoaderHelper helper) {
+		int itemId = Integer.parseInt((String) helper.getKey());
+		String userName = System.getProperty("postgres.username");
```

Comment:
Do the user name and password fetch in the initialize method

Reply:
moved